cloudformation/rds/aws-rds-dbcluster.go

```diff
@@ -95,9 +80,9 @@ type DBCluster struct {
 	EnableIAMDatabaseAuthentication *bool `json:"EnableIAMDatabaseAuthentication,omitempty"`
 
 	// Engine AWS CloudFormation Property
-	// Required: false
+	// Required: true
 	// See: http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-rds-dbcluster.html#cfn-rds-dbcluster-engine
-	Engine *string `json:"Engine,omitempty"`
```

Comment:
This change was unexpected, I believe it is because it changed in the source info, but wanting some input here to make sure

Reply:
I believe sth is wrong here - because http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-rds-dbcluster.html#cfn-rds-dbcluster-engine is saying that this is required field and we are using it in multiple stacks

Reply:
I believe this is file - there is replacement of pointer because filed is required